app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_filter-content.less

```diff
@@ -0,0 +1,61 @@
+.sidebar {
```

Comment:
Please use `& when (@media-common = true) {}` for styles used for both desktop and mobile